### DISCUSSION TITLE
根据文档中的示例，计算teds是结果为0.0

### DISCUSSION THREAD

**Discussion by Rainbow8pony** [PaddlePaddle/PaddleOCR#14352] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

![屏幕截图 2024-12-08 120059](https://github.com/user-attachments/assets/ee6eaef7-6236-4b28-84cf-ab08c1877332)

### 🏃‍♂️ Environment (运行环境)

docker中运行
![屏幕截图 2024-12-08 120617](https://github.com/user-attachments/assets/78ecb796-e5fc-496e-adea-34817160ebd0)

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

![屏幕截图 2024-12-08 120659](https://github.com/user-attachments/assets/01012942-4f53-440f-911f-eeb484128cc0)
zh这一部分，未做改动

**Comment by SWHL** [comment]
单独推理一张图，是否可以出结果呢？
